### PR TITLE
Fix redirects in code doc

### PR DESCRIPTION
**Pull request by RaananW**
Now redirects to the new doc URLs.

### CODE REVIEW

File: packages/dev/core/src/Cameras/VR/vrDeviceOrientationArcRotateCamera.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ Node.AddNodeConstructor("VRDeviceOrientationArcRotateCamera", (name, scene) => {
 
 /**
  * Camera used to simulate VR rendering (based on ArcRotateCamera)
- * @see https://doc.babylonjs.com/babylon101/cameras#vr-device-orientation-cameras
+ * @see https://doc.babylonjs.com/features/featuresDeepDive/cameras#vr-device-orientation-cameras
```
Comment (Popov72) [inline packages/dev/core/src/Cameras/VR/vrDeviceOrientationArcRotateCamera.ts:17]: https://doc.babylonjs.com/features/featuresDeepDive/cameras/camera_introduction#vr-device-orientation-cameras

File: packages/dev/core/src/Cameras/arcRotateCamera.ts [hunk 2]
```diff
@@ -666,14 +666,14 @@ export class ArcRotateCamera extends TargetCamera {
 
     /**
      * Defines whether the camera should check collision with the objects oh the scene.
-     * @see https://doc.babylonjs.com/babylon101/cameras,_mesh_collisions_and_gravity#how-can-i-do-this
+     * @see https://doc.babylonjs.com/features/featuresDeepDive/cameras/camera_collisions#how-can-i-do-this
```
Comment (Popov72) [inline packages/dev/core/src/Cameras/arcRotateCamera.ts:669]: https://doc.babylonjs.com/features/featuresDeepDive/cameras/camera_collisions#how-can-i-do-this-

File: packages/dev/core/src/Cameras/followCamera.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ Node.AddNodeConstructor("ArcFollowCamera", (name, scene) => {
 /**
  * A follow camera takes a mesh as a target and follows it as it moves. Both a free camera version followCamera and
  * an arc rotate version arcFollowCamera are available.
- * @see https://doc.babylonjs.com/features/cameras#follow-camera
+ * @see https://doc.babylonjs.com/features/featuresDeepDive/cameras#follow-camera
```
Comment (Popov72) [inline packages/dev/core/src/Cameras/followCamera.ts:21]: https://doc.babylonjs.com/features/featuresDeepDive/cameras/camera_introduction#followcamera

File: packages/dev/core/src/Cameras/freeCamera.ts [hunk 4]
```diff
@@ -15,13 +15,13 @@ declare type Collider = import("../Collisions/collider").Collider;
 /**
  * This represents a free type of camera. It can be useful in First Person Shooter game for instance.
  * Please consider using the new UniversalCamera instead as it adds more functionality like the gamepad.
- * @see https://doc.babylonjs.com/features/cameras#universal-camera
+ * @see https://doc.babylonjs.com/features/featuresDeepDive/cameras#universal-camera
```
Comment (Popov72) [inline packages/dev/core/src/Cameras/freeCamera.ts:18]: https://doc.babylonjs.com/features/featuresDeepDive/cameras/camera_introduction#universal-camera

File: packages/dev/core/src/Cameras/virtualJoysticksCamera.ts [hunk 5]
```diff
@@ -13,14 +13,14 @@ Node.AddNodeConstructor("VirtualJoysticksCamera", (name, scene) => {
  * This represents a free type of camera. It can be useful in First Person Shooter game for instance.
  * It is identical to the Free Camera and simply adds by default a virtual joystick.
  * Virtual Joysticks are on-screen 2D graphics that are used to control the camera or other scene items.
- * @see https://doc.babylonjs.com/features/cameras#virtual-joysticks-camera
+ * @see https://doc.babylonjs.com/features/featuresDeepDive/cameras#virtual-joysticks-camera
```
Comment (Popov72) [inline packages/dev/core/src/Cameras/virtualJoysticksCamera.ts:16]: https://doc.babylonjs.com/features/featuresDeepDive/cameras/camera_introduction#virtual-joysticks-camera

File: packages/dev/core/src/Engines/engine.ts [hunk 6]
```diff
@@ -754,7 +754,7 @@ export class Engine extends ThinEngine {
 
     /**
      * Gets a boolean indicating that the engine is running in deterministic lock step mode
-     * @see https://doc.babylonjs.com/babylon101/animations#deterministic-lockstep
+     * @see https://doc.babylonjs.com/features/featuresDeepDive/animation#deterministic-lockstep
```
Comment (Popov72) [inline packages/dev/core/src/Engines/engine.ts:757]: https://doc.babylonjs.com/features/featuresDeepDive/animation/advanced_animations#deterministic-lockstep

File: packages/dev/core/src/Materials/Textures/mirrorTexture.ts [hunk 7]
```diff
@@ -15,13 +15,13 @@ import type { UniformBuffer } from "../uniformBuffer";
  * It will dynamically be rendered every frame to adapt to the camera point of view.
  * You can then easily use it as a reflectionTexture on a flat surface.
  * In case the surface is not a plane, please consider relying on reflection probes.
- * @see https://doc.babylonjs.com/how_to/reflect#mirrors
+ * @see https://doc.babylonjs.com/features/featuresDeepDive/materials/using/reflectionTexture#mirrors
```
Comment (Popov72) [inline packages/dev/core/src/Materials/Textures/mirrorTexture.ts:18]: https://doc.babylonjs.com/features/featuresDeepDive/materials/using/reflectionTexture#mirrortexture

File: packages/dev/core/src/Materials/Textures/refractionTexture.ts [hunk 8]
```diff
@@ -4,13 +4,13 @@ import { RenderTargetTexture } from "../../Materials/Textures/renderTargetTextur
 /**
  * Creates a refraction texture used by refraction channel of the standard material.
  * It is like a mirror but to see through a material.
- * @see https://doc.babylonjs.com/how_to/reflect#refraction
+ * @see https://doc.babylonjs.com/features/featuresDeepDive/materials/using/reflectionTexture#refraction
```
Comment (Popov72) [inline packages/dev/core/src/Materials/Textures/refractionTexture.ts:7]: https://doc.babylonjs.com/features/featuresDeepDive/materials/using/reflectionTexture#refractiontexture

File: packages/dev/core/src/Particles/IParticleSystem.ts [hunk 9]
```diff
@@ -614,7 +614,7 @@ export interface IParticleSystem {
     getRampGradients(): Nullable<Array<Color3Gradient>>;
 
     /** Gets or sets a boolean indicating that ramp gradients must be used
-     * @see https://doc.babylonjs.com/babylon101/particles#ramp-gradients
+     * @see https://doc.babylonjs.com/features/featuresDeepDive/particles/particle_system/particle_system_intro#ramp-gradients
```
Comment (Popov72) [inline packages/dev/core/src/Particles/IParticleSystem.ts:617]: https://doc.babylonjs.com/features/featuresDeepDive/particles/particle_system/ramps_and_blends

File: packages/dev/core/src/Particles/solidParticleSystem.ts [hunk 10]
```diff
@@ -58,7 +58,7 @@ export class SolidParticleSystem implements IDisposable {
     public mesh: Mesh;
     /**
      * This empty object is intended to store some SPS specific or temporary values in order to lower the Garbage Collector activity.
-     * Please read : https://doc.babylonjs.com/how_to/Solid_Particle_System#garbage-collector-concerns
+     * Please read : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_intro#garbage-collector-concerns
```
Comment (Popov72) [inline packages/dev/core/src/Particles/solidParticleSystem.ts:61]: https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/optimize_sps#limit-garbage-collection

File: packages/dev/core/src/Particles/solidParticleSystem.ts [hunk 11]
```diff
@@ -70,7 +70,7 @@ export class SolidParticleSystem implements IDisposable {
      * This array is the first element of the pickedBySubMesh array : sps.pickBySubMesh[0].
      * It's not pertinent to use it when using a SPS with the support for MultiMaterial enabled.
      * Use the method SPS.pickedParticle(pickingInfo) instead.
-     * Please read : https://doc.babylonjs.com/how_to/Solid_Particle_System#pickable-particles
+     * Please read : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_intro#pickable-particles
```
Comment (Popov72) [inline packages/dev/core/src/Particles/solidParticleSystem.ts:73]: https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/picking_sps

File: packages/dev/core/src/Particles/solidParticleSystem.ts [hunk 12]
```diff
@@ -743,7 +743,7 @@ export class SolidParticleSystem implements IDisposable {
 
     /**
      * Adds some particles to the SPS from the model shape. Returns the shape id.
-     * Please read the doc : https://doc.babylonjs.com/how_to/Solid_Particle_System#create-an-immutable-sps
+     * Please read the doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_intro#create-an-immutable-sps
```
Comment (Popov72) [inline packages/dev/core/src/Particles/solidParticleSystem.ts:746]: https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/immutable_sps

File: packages/dev/core/src/Particles/solidParticleSystem.ts [hunk 13]
```diff
@@ -1757,7 +1757,7 @@ export class SolidParticleSystem implements IDisposable {
     }
     /**
      * Visibility helper : Recomputes the visible size according to the mesh bounding box
-     * doc : https://doc.babylonjs.com/how_to/Solid_Particle_System#sps-visibility
+     * doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_intro#sps-visibility
```
Comment (Popov72) [inline packages/dev/core/src/Particles/solidParticleSystem.ts:1760]: https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_visibility

File: packages/dev/core/src/Particles/solidParticleSystem.ts [hunk 14]
```diff
@@ -1979,14 +1979,14 @@ export class SolidParticleSystem implements IDisposable {
     /**
      * This function does nothing. It may be overwritten to set all the particle first values.
      * The SPS doesn't call this function, you may have to call it by your own.
-     * doc : https://doc.babylonjs.com/how_to/Solid_Particle_System#particle-management
+     * doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_intro#particle-management
```
Comment (Popov72) [inline packages/dev/core/src/Particles/solidParticleSystem.ts:1982]: https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/manage_sps_particles

File: packages/dev/core/src/Particles/solidParticleSystem.ts [hunk 15]
```diff
@@ -2012,7 +2012,7 @@ export class SolidParticleSystem implements IDisposable {
      * @param particle the current particle
      * @param vertex the current vertex of the current particle : a SolidParticleVertex object
      * @param pt the index of the current vertex in the particle shape
-     * doc : https://doc.babylonjs.com/how_to/Solid_Particle_System#update-each-particle-shape
+     * doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_intro#update-each-particle-shape
```
Comment (Popov72) [inline packages/dev/core/src/Particles/solidParticleSystem.ts:2015]: https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_vertices

**Review by Popov72 — CHANGES_REQUESTED** [review]
When a link was wrong I gave the corrected one, but I did it only for the first occurrence: the link generally appears multiple times in the same or different files.

I definitely missed some :)

File: packages/dev/core/src/Particles/solidParticleSystem.ts (hunk 10 above)
Comment (RaananW) [reply packages/dev/core/src/Particles/solidParticleSystem.ts:61]:
```suggestion
     * Please read : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/optimize_sps#limit-garbage-collection
```

File: packages/dev/core/src/Particles/solidParticleSystem.ts (hunk 11 above)
Comment (RaananW) [reply packages/dev/core/src/Particles/solidParticleSystem.ts:73]:
```suggestion
     * Please read : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/picking_sps
```

File: packages/dev/core/src/Particles/solidParticleSystem.ts (hunk 12 above)
Comment (RaananW) [reply packages/dev/core/src/Particles/solidParticleSystem.ts:746]:
```suggestion
     * Please read the doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/immutable_sps
```

File: packages/dev/core/src/Particles/solidParticleSystem.ts (hunk 13 above)
Comment (RaananW) [reply packages/dev/core/src/Particles/solidParticleSystem.ts:1760]:
```suggestion
     * doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_visibility
```

File: packages/dev/core/src/Particles/solidParticleSystem.ts (hunk 14 above)
Comment (RaananW) [reply packages/dev/core/src/Particles/solidParticleSystem.ts:1982]:
```suggestion
     * doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/manage_sps_particles
```

File: packages/dev/core/src/Particles/solidParticleSystem.ts (hunk 15 above)
Comment (RaananW) [reply packages/dev/core/src/Particles/solidParticleSystem.ts:2015]:
```suggestion
     * doc : https://doc.babylonjs.com/features/featuresDeepDive/particles/solid_particle_system/sps_vertices
```

**Review by Popov72 — APPROVED** [review]

**Pull request merged**